server/routes/case.js

```diff
@@ -16,7 +16,13 @@ router.all(['/:caseId/stage/:stageId', '/:caseId/stage/:stageId/allocate'],
     caseSummaryMiddleware,
     getDocumentList,
     getCaseNotes);
-router.all(['/:caseId/stage/:stageId/entity/:entity/:context/:action', '/:caseId/stage/:stageId/entity/:entity/:action'], getFormForCase, hydrateFields);
+
+router.all(['/:caseId/stage/:stageId/entity/:entity/:context/:action',
+    '/:caseId/stage/:stageId/entity/:entity/:context/:caseType/:action',
+    '/:caseId/stage/:stageId/entity/:entity/:action'
+], getFormForCase, hydrateFields
+);
+
```

Comment:
Again... slight formatting fix. It just makes it easier to read for future FOI team

Reply:
The linter keeps complaining when I try anything else and 'corrects' it to this format.

Reply:
no worries... leave it as it is then